Build-your-first-dApp.md

```diff
@@ -128,46 +148,58 @@ OPTIONAL: Inside the `<head>` tag, add some styles to make it look nicer
 
 Now it's time to create a Solidity smart contract.
 
-1. You can use any editor you like to make the contract. For this tutorial we recommend the online IDE [Remix](https://remix.ethereum.org)
+1. You can use any editor you like to make the contract. For this part of the tutorial we recommend the online IDE [Remix](https://remix.ethereum.org)
 2. Go to [Remix](https://remix.ethereum.org)
 3. Check out the "Solidity Compiler", and "Deploy and Run Transactions" tabs. If they are not present, enable them in the plugin manager
 4. Create a new solidity file in remix, named `mood.sol`
 5. Write the contract
 
    - Specify the solidity version and add a license
-
-   ```
-   // SPDX-License-Identifier: MIT
-    pragma solidity ^0.8.1;
-   ```
-
+    
+    <p>
+
+      ```JS
+      // SPDX-License-Identifier: MIT
+      pragma solidity ^0.8.1;
+      ```
+    </p>
+  
    - Define the contract
+ 
+    <p>
 
-   ```
-    contract MoodDiary{
-
-    }
-   ```
+      ```JS
+      contract MoodDiary{
+        // This is the contract's body, here you'll specify the logic for this contract.
+      }
+      ```
+    </p>
 
    - Inside the contract create a variable called mood
+ 
+    <p>
 
-   ```
-    string mood;
-   ```
+      ```
+      string mood;
+      ```
+    </p>
 
    - Next, create Read and Write functions
-
-   ```
-    //create a function that writes a mood to the smart contract
-    function setMood(string memory _mood) public{
-        mood = _mood;
-    }
-
-    //create a function the reads the mood from the smart contract
-    function getMood() public view returns(string memory){
-        return mood;
-    }
-   ```
+ 
+    <p>
```

Comment:
This should work, but anyway we could replace any HTML with Markdown?

Reply:
No idea, I don't know any other ways of vertical spacing on markdown, I'm also not an expert using MD with Next, so if you have any idea let me know.

Reply:
The good ol' enter spaces should do actually, or maybe a [Horizontal Rule](https://www.markdownguide.org/cheat-sheet/)?

Reply:
Horizontal rules don't look nice, since it adds a separator between the description of the highlighted code, and the code. Enter spaces behave the same as default spaces, the code is too close to the description.

I'll try something else, and if it doesn't work, I'll just stick enter spaces for the sake of it. This PR is taking too long...

Reply:
Honestly, it's weird that there doesn't seem to be any "native" md vertical spacer unless you use `<br>`'s or the HTML entity `&nbsp;``, so I'm going to stick with the latter I guess.

Reply:
Can you try it out, and validate as well that it works on next?

Also, thanks for your time :) 👍🏼

Reply:
No worries, I'm not sure whether this will work, but I'll push and if not, revert.